app/containers/Search/index.js

```diff
@@ -38,17 +38,24 @@ export const SearchContainerComponent = props => {
 
   const onFocusOut = event => {
     const { relatedTarget } = event;
-    // Fix for browser where relatedTarget is `null`
-    // Maybe a better overall solution would be to use React's `onFocus` and `onBlur` vs. refs + focusout
-    if (relatedTarget == null) return;
+
     const { current: input } = inputRef;
     const { current: suggest } = suggestRef;
 
     const blurFromInput = !input || (input !== null && !input.contains(relatedTarget));
     const blurFromSuggest = !suggest || (suggest !== null && !suggest.contains(relatedTarget));
 
     if (blurFromInput && blurFromSuggest) {
-      setShowSuggest(false);
+      // Fix for Safari, Safari iOS
+      // ==========================
+      // In Safari the focusout event is triggered before
+      // the click is handled. And because the links are hidden,
+      // they are not followed.
+      // Maybe a better overall solution would be
+      // to use React's `onFocus` and `onBlur` vs. refs + focusout.
+      // Cause this solution is still subject to a possible
+      // race condition, although 200ms seems more than fine.
+      delay(() => setShowSuggest(false), 200);
```

Comment:
I'm not sure if I like this solution. Using delays to solve this stuff might work in some cases, but it's hard/impossible to predict if this will solve the problem on other machines (slower ones for example).

Reply:
I agree. Maybe we better don't merge this request. And only rollback the `if (relatedTarget == null) return;` on line 43. Some documentation of this bug and what is causing it is needed though. So it can be picked up when refactoring this code / component in the future.